packages/rps/src/redux/game-v2/__tests__/scenarios.ts

```diff
@@ -0,0 +1,110 @@
+import {bigNumberify} from 'ethers/utils';
+import {AppData, RoundProposed, Start, RoundAccepted, Reveal, hashPreCommit} from '../../../core';
+import {ChannelState, Result, Weapon, ChannelStatus} from '../../../core';
+import {LocalState} from '../state';
+import * as s from '../state';
+
+export const channelId = '0xabc234';
+export const aName = 'Alice';
+export const bName = 'Bob';
+const aUserId = 'userA';
+const bUserId = 'userB';
+const aDestination = 'destinationA';
+const bDestination = 'destinationB';
+export const aAddress = '0x5409ED021D9299bf6814279A6A1411A7e866A631';
+export const bAddress = '0x6Ecbe1DB9EF729CBe972C83Fb886247691Fb6beb';
+export const stake = bigNumberify(1);
+const roundBuyIn = stake;
+export const aWeapon = Weapon.Rock;
+const playerAWeapon = aWeapon;
+export const salt = '0x' + '4'.repeat(64);
+const preCommit = hashPreCommit(aWeapon, salt);
+export const bWeapon = Weapon.Scissors;
+const playerBWeapon = bWeapon;
+
+const appData = {
+  start: {type: 'start'} as Start,
+  roundProposed: {type: 'roundProposed', stake, preCommit} as RoundProposed,
+  roundAccepted: {type: 'roundAccepted', stake, preCommit, playerBWeapon} as RoundAccepted,
+  reveal: {type: 'reveal', playerAWeapon, playerBWeapon, salt} as Reveal,
+};
```

Comment:
I don't think you want to use type assertions here. You could do
```
const appData: {
  start: Start,
  roundProposed: RoundProposed,
  roundAccepted: RoundAccepted,
  reveal: Reveal,
  } = {
  start: {type: 'start'},
  roundProposed: {type: 'roundProposed', stake, preCommit},
  roundAccepted: {type: 'roundAccepted', stake, preCommit, playerBWeapon},
  reveal: {type: 'reveal', playerAWeapon, playerBWeapon, salt},
};
```
See [here](https://basarat.gitbooks.io/typescript/docs/types/type-assertion.html)

> If you forget a property, the compiler will not complain